server/db/model.py

```diff
@@ -0,0 +1,24 @@
+from pymodm import MongoModel, fields
+
+
+class User(MongoModel):
+    """
+    User model
+
+
+    Example:
+        from pymodm.connection import connect
+        connect("mongodb://localhost:8000/db", alias="user-db")
+        new_user=User("a@b.com",first_name="John",last_name="Smith",salted_password="aa-bb-cc-dd").save()
+
+    """
```

Comment:
Not sure if required, but would we want to add an "id" field which can be looked for on the front end to verify that the user info is coming from the database, rather than the form?

Reply:
I think this is a great point that I overlooked - given that this is MongoDB, the primary_key will always be called _id and by setting email as primary_key, that means that _id's will be emails (see https://pymodm.readthedocs.io/en/stable/api/ and search primary_key)

I personally think it is more scalable down the road to have _id's to be the autogenerated MongoDB ObjectIDs and to have email as just an indexed field 

I just think it'd be easier to abstract the _id from the email, in case users want to change emails, etc. it'll be a lot easier to deal with